src/renderer/component/common/form-components/form-field.jsx

```diff
@@ -15,29 +19,30 @@ type Props = {
   defaultValue?: string | number,
   placeholder?: string | number,
   children?: React.Node,
-  stretch?: boolean
+  stretch?: boolean,
 };
 
 export class FormField extends React.PureComponent<Props> {
   render() {
     const { render, label, prefix, postfix, error, helper, name, type, children, stretch, ...inputProps } = this.props;
 
-    // Allow a type prop to determine the input or more customizability with a render prop
     let Input;
```

Comment:
Why capital? To represent a JSX element?

Reply:
It was capital when I had the render function. Now that I removed it I'll set it back to lowercase.